src/plugins/intel_gpu/include/intel_gpu/graph/network.hpp

```diff
@@ -223,6 +225,8 @@ struct network {
     /* Common memory pointer for shape_info */
     memory::ptr _shape_info_ptr;
 
+    memory::ptr _output_remote_mem_ptr = nullptr;
```

Comment:
We can use unordered_map here, as network may have multiple outputs with assigned remote tensors

Reply:
Update to handle multiple output tensors